packages/ibm-products-styles/src/__tests__/__snapshots__/styles.test.js.snap

```diff
@@ -1908,8 +1908,14 @@ p.c4p--about-modal__copyright-text:first-child {
   letter-spacing: var(--cds-label-01-letter-spacing, 0.32px);
 }
 
-.c4p--create-full-page .c4p--create-full-page {
+.c4p--create-full-page {
+  display: grid;
+  height: 100%;
   color: var(--cds-text-primary, #161616);
+  grid-template-rows: minmax(auto, 100%);
```

Comment:
Hello @lee-chase 

This doesn't work on some browsers, causing some height issues as you can see from the screenshot.

![Screenshot 2024-09-02 at 15 40 42](https://github.com/user-attachments/assets/cf076934-0858-482c-8288-83b9313f12d1)

Reply:
I have a fix available by using `auto minmax(min-content, 100%)` instead... if there are no objections, I can contribute that.